[BUG] Module not found when using latest version

### Are you using the latest version of this library?

- [X] I verified that the issue exists in the latest next-safe-action release

### Is there an existing issue for this?

- [X] I have searched the existing issues and found nothing that matches

### Describe the bug

After updating from 7.1.3 to 7.2.1 I get the following error:

```
 ⨯ ./node_modules/next-safe-action/dist/index.mjs:1:1
Module not found: Can't resolve './adapters/zod'
> 1 | import { zodAdapter } from "./adapters/zod";
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  2 | import { SafeActionClient } from "./safe-action-client";
  3 | import { DEFAULT_SERVER_ERROR_MESSAGE } from "./utils";
  4 | import {
```

### Reproduction steps

Server action with zod schema. 

### Expected behavior

No error.

### Minimal reproduction example

Private repo

### Operating System

Windows 11

### Library version

v7.2.1

### Next.js version

15.0.0-canary.72

### Additional context

_No response_

--- a/packages/next-safe-action/tsup.config.ts
+++ b/packages/next-safe-action/tsup.config.ts
@@ -1,8 +1,8 @@
 import { defineConfig } from "tsup";
 
 export default defineConfig({
-	entry: ["src/*.ts", "src/adapters/*.ts"],
-	bundle: false,
+	entry: ["src/index.ts", "src/hooks.ts", "src/stateful-hooks.ts", "src/adapters/*.ts"],
+	bundle: true,
 	format: ["esm"],
 	clean: true,
 	splitting: false,

